scripts/sftp_upload.py

```diff
@@ -0,0 +1,23 @@
+# :coding: utf-8
+# :copyright: Copyright (c) 2014-2022 ftrack
+
+import ftrack_api.structure.id
+from sftp import SFTPAccessor
+import os
+
+
+session = ftrack_api.Session()
+location = session.query('Location where name is "studio.sftp"').one()
```

Comment:
@iwootten 

might worth replacing it with

`location = session.ensure('Location', {"name":"studio.sftp"})`